Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -379,6 +389,115 @@ public final class SwiftTargetBuildDescription {
                 }
             }
 
+            // FIXME: Copied from `Lock.swift` in TSCBasic, would be nice if we had a better way
```

Comment:
I wonder if we can symlink or copy the files instead?

Reply:
ie maybe all these files are copied from "templates" instead of being stringified version, which may also solve the copy/pasta 🍝

Reply:
Yah I was thinking about making these files that we ship with SwiftPM, but we have to locate them of course etc. 

I think that could be a good way to do it if we had access to the `.embedInCode` feature for resources, so maybe we just leave them as strings for now and then use that in the next release or whenever we can upgrade to a 5.9 tools-version.

There's also the issue that these aren't 1:1 copies, especially in the case of `FileLock` because we can't introduce even further dependencies on TSC stuff here.